DOCS-162 - Add new UI hidden instructions to Metrics (Kanso) (#4065)
SumoLogic/sumologic-documentation: docs/metrics/manage-metric-volume/data-limits-for-metrics.md

It is important to understand the volume of metrics that you are ingesting into Sumo Logic.

For billing and reporting purposes, data volume for metrics is measured in Data Points per Minute (DPM). When the DPM limit is exceeded, data is cached on the host and the Source is throttled. To check your current account usage relative to your available DPM limit, see the **Account Overview** tab.  

<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  To access the **Account Overview** tab, in the main Sumo Logic menu select **Administration > Account**. 

<!--Kanso 

> [**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the **Account Overview** tab, in the top menu select **Administration** and then under **Account** select **Account Overview**. You can also click the **Go To...** menu at the top of the screen and select **Account Overview**. 

Reviewer: ```suggestion
[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the **Account Overview** tab, go to the top menu select **Administration**, and then under **Account**, select **Account Overview**. Alternatively, you can click the **Go To...** menu at the top of the screen and select **Account Overview**. 
```
